department-of-veterans-affairs/abd-vro
# Mcp 1990 - Show empty list for medical evidence item when there is no record found in Lighthouse Fhir service.

The current medical evidence data return from Lighthouse Patient Health data query does not list the item if there is no data from the Lighthouse query. MCP-1990 requires to list all the items and set an empty list for the item with no Lighthouse data.

## What was the problem?


Associated tickets or Slack threads:

MCP-1990

## How does this fix it?[^1]

Show empty list for the item with no data found in Lighthouse.

## How to test this PR
- Step 1. Build the application 
- Step 2. Use the case listed in MCP-1990 to test /health-daa-assessment endpoint. You should see empty lists for some medical evidence items.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
